test/e2e/framework/framework.go

```diff
@@ -394,15 +394,15 @@ func (f *Framework) NewNginxStatefulSet(cluster framework.ClusterIndex) *appsv1.
 					Containers: []v1.Container{
 						{
 							Name:            statefulServiceName,
-							Image:           "quay.io/testing-farm/nginx:latest",
+							Image:           "localhost:5000/nettest:local",
```

Comment:
This assumes that it was loaded to a local registry which might not be true when calling `subctl verify`, better use `quay.io/submariner/nettest:devel`